### PR TITLE
Added icon for Clear Amulet

### PR DESCRIPTION
**Pull request by PacFire**
I have created an icon for the Clear Amulet item since the expansion didn't have it.
sprite was fully created by me
(this is also my first time doing a PR!)
link to the .png: https://cdn.discordapp.com/attachments/1112500679009898498/1123001295876395059/clear_amulet.png
link to the .pal: https://cdn.discordapp.com/attachments/1112500679009898498/1123001296132243466/clear_amulet.pal

## Images
image

## **Discord contact info**
pacfire

### CODE REVIEW

File: include/graphics.h [hunk 1]
```diff
@@ -8550,6 +8550,8 @@ extern const u32 gItemIcon_RoomService[];
 extern const u32 gItemIconPalette_RoomService[];
 extern const u32 gItemIcon_UtilityUmbrella[];
 extern const u32 gItemIconPalette_UtilityUmbrella[];
+extern const u32 gItemIcon_ClearAmulet[];
+extern const u32 gItemIconPalette_ClearAmulet[];
```
Comment (AsparagusEduardo) [inline include/graphics.h:8554]: Please move this to be next to the other Gen 9 items.

Comment (PacFire) [reply include/graphics.h]: done

Comment (PacFire) [reply include/graphics.h]: my bad for not noticing that :'D

Comment (AsparagusEduardo) [reply include/graphics.h]: Don't worry, I also noticed just now that the order differs between them, but that can be cleaned up later xD

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**